<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSimplificacion</class>
 <widget class="QDialog" name="DialogSimplificacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupSimplificacion">
     <property name="title">
      <string>Métodos de simplificación</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkTautologicas">
        <property name="text">
         <string>Clausulas tautológicas</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkEquivalentes">
        <property name="text">
         <string>Clausulas equivalentes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkLiteralesPuros">
        <property name="text">
         <string>Literales puros</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="botonSimplificar">
        <property name="text">
         <string>Simplificar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupText">
     <property name="title">
      <string>Conjunto simplificado</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="textoSimplificado">
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botonSimplificar</sender>
   <signal>clicked()</signal>
   <receiver>DialogSimplificacion</receiver>
   <slot>simplificarConjunto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>simplificarConjunto()</slot>
 </slots>
</ui>
